test/src/module/storage/s3Test.c

```diff
@@ -200,7 +200,11 @@ testS3Server(void)
 
```

Comment:
Just a question since I don't see any common expressions in regExpPrefix nor here, but a common expression would be `^[0-F]{16}$` so I just want to confirm that the regExpPrefix would return NULL but the s3Test would still filter the return based on the expression, yes?

Reply:
The filtering happens in storageInfoList(), but yes.